src/common/secret/Cargo.toml

```diff
@@ -17,7 +17,7 @@ parking_lot = { workspace = true }
 prost = { workspace = true }
 reqwest = { workspace = true }
 risingwave_pb = { workspace = true }
-serde = { version = "1" }
+serde = { version = "1", features = ["derive"] }
```

Comment:
Shall we move the depended crate declaration to the root workspace `Cargo.toml`, and change the dependency in the `Cargo.toml` in all concrete crates to `serde = { workspace = true }`?